test/api/test_customer.py

```diff
@@ -480,3 +509,43 @@ def test_createContact(self, mock_requests):
         self.assertEqual(mock_requests.last_request.qs, {})
         self.assertEqual(mock_requests.last_request.json(), createContact)
         self.assertTrue(isinstance(expected, Contact))
+
+    @requests_mock.mock()
+    def test_notes(self, mock_requests):
+        mock_requests.register_uri(
+            "GET",
+            "https://api.chartmogul.com/v1/customer_notes?customer_uuid=cus_00000000-0000-0000-0000-000000000000",
+            status_code=200,
+            json=allNotes,
+        )
+
+        config = Config("token")
+        notes = Customer.notes(config, uuid="cus_00000000-0000-0000-0000-000000000000").get()
```

Comment:
Could you also test with other params? e.g. `per_page`, `cursor`. Reason why I asked is because you defined the URL here:

https://github.com/chartmogul/chartmogul-python/blob/2170a939238d61f7392f9619b26b3452fd68fecc/chartmogul/api/customer.py#L86

and the `customer_uuid` is part of the URL. I'm not sure if the parameters would merge correctly with the rest (i.e. would it look like `?customer_uuid=nn&cursor=nn` or `?customer_uuid=nn?cursor=nn`), so maybe it's better to test that